getting-started/kubernetes/self-managed-public-cloud/aws.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Self-managed Kubernetes in Amazon Web Services (AWS)
+description: Use Calico with a self-managed Kubernetes cluster in Amazon Web Services (AWS).
+---
+
+### Big picture
+
+Use {{site.prodname}} with a self-managed Kubernetes cluster in Amazon Web Services. 
+
+### Value
+
+Managing your own Kubernetes cluster (as opposed to using a managed-Kubernetes service like EKS), gives you the most flexibility in configuring {{site.prodname}} and Kubernetes. {{site.prodname}} provides both **networking** and **network security** for containers, virtual machines, and native host-based workloads across a broad range of platforms including Kubernetes, OpenShift, Docker EE, OpenStack, and bare metal services. {{site.prodname}} combines flexible networking capabilities with "run-anywhere" security enforcement to provide a solution with native Linux kernel performance and true cloud-native scalability.
+
+### Concepts
+
+Kubernetes Operations (kops) is a cluster management tool that handles provisioning cluster VMs and installing Kubernetes. It has built-in support for using {{site.prodname}} as the Kubernetes networking provider.
+
+### Before you begin...
+
+- Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Install [AWS CLI tools](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
+
+### How to
+
+There are many ways to install and manage Kubernetes in AWS. Using Kubernetes Operations (kops) is a good default choice for most people, as it gives you access to all of {{site.prodname}}’s [flexible and powerful networking features]({{site.baseurl}}/networking). However, there are other options that may work better for your environment.
+
+- [Kubernetes Operations for Calico networking and network policy](#kubernetes-operations-for-calico-networking-and-network-policy)
+- [Other options and tools](#other-options-and-tools)
+
+#### Kubernetes Operations for Calico networking and network policy
+
+To use kops to create a cluster with {{site.prodname}} networking and network policy:
+
+1. [Install kops](https://kops.sigs.k8s.io/install/) on your workstation.
+1. [Set up your environment for AWS](https://kops.sigs.k8s.io/getting_started/aws/).
+1. Configure kops to use {{site.prodname}} for networking.  
+   The easiest way to do this is to pass `--networking calico` to kops when creating the cluster. For example
+
+   ```
+   kops create cluster \
+    --zones us-west-2a \
+    --networking calico \
+    name-of-your-cluster
+   ```
```

Comment:
This doesn't work, at least with kops `Version 1.15.0 (git-9992b4055)`. I get the following:

```
$    kops create cluster \
>     --zones us-west-2a \
>     --networking calico \
>     name-of-your-cluster

State Store: Required value: Please set the --state flag or export KOPS_STATE_STORE.
For example, a valid value follows the format s3://<bucket>.
You can find the supported stores in https://github.com/kubernetes/kops/blob/master/docs/state.md.
```

Reply:
@lmm You need to set up S3 and export the `KOPS_STATE_STORE` as described on: https://kops.sigs.k8s.io/getting_started/aws/

I've updated the second item above to hopefully make this a bit more clear. What do you think?

Reply:
Looks good, thanks!